asistente_ladm_col/logic/quality_rules/qr_administrative_source_with_duplicate_records.py

```diff
@@ -0,0 +1,107 @@
+"""
```

Comment:
I think we can use for naming quality rules files something like qr_administrative_source_duplicate_records.py instead of use 'with' which makes the name larger. What do you think?

Reply:
I think it is a good idea, change made in commit  281842d